README.md

```diff
@@ -1,2 +1,61 @@
 # next-runtime-env
-NextJS plugin to pass environment variables to NextJS's runtime configuration
+NextJS plugin to pass environment variables to NextJS's runtime configuration.
+
+By default, NextJS does not make available `process.env` to the client side. The available solution for this problem is to configure a runtime config in `next.config.js` to pass values to either server and client or just server.
+
+The way to do it is by passing values to the `publicRuntimeConfig` and `serverRuntimeConfig` properties of the NextJS' config.
+
+This plugin does just that.
+
+## Installation
+
+```sh
+$ npm i -D @moxy/next-runtime-env
+```
+
+This library is written and published in modern JavaScript. If you target older versions of node please make sure to transpile accordingly.
+
+## Usage
+
+```js
+// next.config.js
+const withRuntimeEnv = require('@moxy/next-runtime-env');
+
+module.exports = withRuntimeEnv();
+```
+
+Multiple configurations can be combined together with function composition. For example:
+
+```js
+// next.config.js
+const withCSS = require('@zeit/next-css');
+const withRuntimeEnv = require('@moxy/next-runtime-env');
+
+module.exports = withCSS(
+    withRuntimeEnv()({
+        cssModules: true, // this options will be passed to withCSS plugin through nextConfig
+    }),
+);
+```
+
+### API
+
+| Option         | Description                                                        | Type    | Default   |
+|----------------|--------------------------------------------------------------------|---------|-----------|
```

Comment:
Nifty pick: you don't need to define `---------------------------` with exact length. It's preferable to use it normally to avoid having have 4 diff lines when a new option is added, that has a larger length in either the name or description. 

e.g.: https://raw.githubusercontent.com/moxystudio/babel-preset-moxy/master/README.md and check the table there

Reply:
Yeah, I was using an online tool to build tables and by default it defined the number of `-` with exact length. I also prefer without it since when changing in the future, it would probably not be correct.

Removed the unnecessary `-`.